kubelet/datadog_checks/kubelet/data/conf.yaml.example

```diff
@@ -1,49 +1,61 @@
 init_config:
 
 instances:
-  - ###
-    ### Common additional tags for all metrics from this check
-    ###
-    #
-    # tags:
-    #   - 'mytag1:myValue1'
-    #
-    ###
-    ### 1.7.6+ clusters expose container metrics in the prometheus format.
-    ### This is the default setting. See next section for legacy clusters.
-    ###
-    #
-    # DEPRECATED: please specify `cadvisor_metrics_endpoint` instead
-    # metrics_endpoint: http://10.8.0.1:10255/metrics/cadvisor
-    #
-    # url of the cadvisor metrics prometheus endpoint
-    # Pass an empty string, or set the cadvisor_port option to disable
-    # cadvisor metrics collection
-    # cadvisor_metrics_endpoint: http://10.8.0.1:10255/metrics/cadvisor
-    #
-    # url of the kubelet metrics prometheus endpoint
-    # Pass an empty string to disable kubelet metrics collection
-    # kubelet_metrics_endpoint: http://10.8.0.1:10255/metrics
-    #
-    # The histogram buckets can be noisy and generate a lot of tags.
-    # send_histograms_buckets controls whether or not you want to pull them.
-    #
-    # send_histograms_buckets: True
-    #
-    ###
-    ### Metric collection for legacy (< 1.7.6) clusters via the kubelet's
-    ### cadvisor port.
-    ### This port is closed by default on k8s 1.7+ and OpenShift, enable it
-    ### via the `--cadvisor-port=4194` kubelet option.
-    ###
-    #
-    # Port to connect to, uncomment and set accordingly to enable collection.
-    # cadvisor_port: 4194
-    #
-    # Whitelist of metrics to collect from cadvisor, these are the default
-    #
-    # enabled_rates:
-    #   - cpu.*
-    #   - network.*
-    # enabled_gauges:
-    #   - filesystem.*
+
+  ## @param tags  - list of key:value element - optional 
+  ## List of tags to attach to every metric, event and service check emitted by this integration.
+  ## 
+  ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+  #
+  #  tags:
+  #    - <KEY_1>:<VALUE_1>
+  #    - <KEY_2>:<VALUE_2>
+
+  ## 1.7.6+ clusters expose container metrics in the prometheus format.
+  ## This is the default setting. See next section for legacy clusters.
+
+  ## @param metrics_endpoint - string - optional
+  ## DEPRECATED: please specify `cadvisor_metrics_endpoint` instead
+  #
+  #  metrics_endpoint: http://10.8.0.1:10255/metrics/cadvisor
```

Comment:
Shall we just remove it from the example config, as `cadvisor_metrics_endpoint` supersedes it since 6.4?